trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative './base_decorator'
+
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    @nameable.correct_name[1...10] if @nameable.correct_name.length > 10
+  end
```

Comment:
**[Optional]**

- [ ] It would be nice if line 5 should have **[0...9]** not **[1...10]** because everything in string or array starts with a zero index so be careful while manipulating arrays and strings 😊